help/destinations/catalog/email-marketing/salesforce-marketing-cloud-exact-target.md

```diff
@@ -213,18 +215,12 @@ All [!DNL Adobe Experience Platform] destinations are compliant with data usage
 ## Errors and troubleshooting {#errors-and-troubleshooting}
 
 ### Unknown errors encountered while pushing events to Salesforce Marketing Cloud {#unknown-errors}
+When checking a dataflow run, if you obtain the following error message: `Unknown errors encountered while pushing events to the destination. Please contact the administrator and try again.`
```

Comment:
Minor wording suggestion: 

When checking a dataflow run, you might encounter the following error message: